src/docs/integrations/slack/index.mdx

```diff
@@ -79,17 +68,19 @@ Add the following scopes to __User Scopes__:
 * `users:read`
 * `users:read.email`
 
-Click __Save Changes__.
-
-_Note: You have the option to set Restrict API Token Usage on this page if you want to ensure your Slack is the only one talking to your Sentry instance._
+<Alert level="info">
+You have the option to set Restrict API Token Usage on this page if you want to ensure your Slack is the only one talking to your Sentry instance.
```

Comment:
nit: Can you put single quotes, or italics around 'Restrict API Token Usage' so it is visually distinctive from the rest of the sentence

Reply:
I think this phrase is missing a word, idk what word tho

> ensure your Slack _____ is the only one

IDK if workspace, or app, or bot makes the most sense. You probably have the most context here.